build.sbt

```diff
@@ -108,20 +113,3 @@ lazy val dynosaur = project
     commonSettings
   ).dependsOn(awssdk)
 
-lazy val scanamo = project
```

Comment:
Looks like this PR removes Scanamo codec support. Should the docs be updated accordingly? 🤔